packages/browser-extension/source/cloneElements.ts

```diff
@@ -0,0 +1,95 @@
+import { Project } from "ts-morph"
```

Comment:
lets rename the file from `source/cloneElements.ts` to `source/react-router/cloneRoute.ts`